step1_build_jars.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
```

Comment:
You just run mvn install, why do you need this file?

Reply:
Same format with other scripts

Reply:
Move then all to `bin/` directory, and name them in more adequate way